server/controllers/finance/reports/invoices/receipt.handlebars

```diff
@@ -132,32 +132,68 @@
       <table style="width: 100%;" class="table-report">
         <tr>
           <td>
-            <h1>{{translate 'FORM.LABELS.TOTAL'}} : </h1>
+            <b>{{translate 'FORM.LABELS.TOTAL'}} : </b>
           </td>
           <td class="text-right">
-            <h3><b>{{currency cost metadata.enterprise.currency_id}}</b></h3>
+            <h5><b>{{currency cost metadata.enterprise.currency_id}}</b></h5>
           </td>
         </tr>
         <tr>
           <td colspan='2' class="text-right">
-            <h4><b> {{numberToText cost lang metadata.enterprise.currencyName}}</b></h4>
+            <em>{{numberToText cost lang metadata.enterprise.currencyName}}</em>
           </td>
         </tr>
 
+        <!-- Prepayment inside invoice -->
+        {{#if prepaymentInsideInvoice}}
```

Comment:
It looks like we have enough space - could we put both currencies on the same line?  That way, we don't waste vertical space.  Something like:

| | | | 
|:---:| ----:| -----:|
| total | 86.00$ | 139320FC |

Reply:
@jniles do we need the total amount in full letter (e.g. Dix mille dollars) ?